daemon/config/config.go

```diff
@@ -116,6 +116,14 @@ type Config struct {
 	DefaultNamespace string `json:"default-namespace,omitempty"`
 }
 
+// GetCgroupDriver gets cgroup driver used in runc.
+func (cfg *Config) GetCgroupDriver() string {
+	// current pouchd only supports directly managing cgroupfs.
+	// TODO: add 'systemd' to make systemd manage cgroupfs rather than directly using it.
+	// In the future we will support this config in the daemon configuration.
+	return "cgroupfs"
```

Comment:
Now we only support cgroupfs, systemd may put into consider,  can you try to get this type from somewhere?

Reply:
> can you try to get this type from somewhere

Could you explain a little bit more about the above sentence? @Ace-Tang Thanks a lot.

Reply:
Not hardcode cgroup driver.

Reply:
OK, I see. Thanks a lot.